tool/ood-gen/lib/tutorial.ml

```diff
@@ -120,6 +120,23 @@ let toc ?(start_level = 1) ?(max_level = 2) doc =
     invalid_arg "toc: ~max_level must be >= ~start_level";
   create_toc ~max_level start_level (headers doc)
 
+exception Missing_Tutorial of string
+
+let any_recommended_next_tuts_are_missing_exn all =
```

Comment:
PR #1965 will add prerequisites tutorials. Those list will require the same check. Let's name this accordingly.
```suggestion
let check_tutorial_references all =
```